webviz_config/_theme_class.py

```diff
@@ -17,8 +17,8 @@ def __init__(self, theme_name: str):
             "style-src": ["'self'", "'unsafe-inline'"],  # [1]
             "script-src": [
                 "'self'",
-                "'unsafe-eval'",  # [2]
-            ],
+                "'unsafe-eval'",
+            ],  # [2]
```

Comment:
Hmm, `black` doesn't complain with current `master` - so we maybe don't have to include this diff here?